src/mcp_atlassian/server.py

```diff
@@ -347,6 +347,11 @@ async def list_tools() -> list[Tool]:
                                 "description": "Whether to include page metadata such as creation date, last update, version, and labels",
                                 "default": True,
                             },
+                            "markdown": {
```

Comment:
```suggestion
                            "convert_to_markdown": {
```
Consider renaming the parameter from `markdown` to something more descriptive like `convert_to_markdown` (to match the internal parameter) or `use_markdown_format` to make its purpose clearer.

Reply:
Good idea, will change that.